### DISCUSSION TITLE
improve the "Tags" creations

### DISCUSSION THREAD

**Discussion by kucingkembar** [stashapp/stash#4744] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
yes it is related to the problem that I asked previously at 
https://github.com/stashapp/stash/issues/4718

**Describe the solution you'd like**
I like this feature is improved:
![Untitled](https://github.com/stashapp/stash/assets/47294266/7b3476fe-e792-4bc9-95fa-add720027291)
the main feature of this software is auto tag "Performers", "Studio", and "Tags",
but somehow the user must "Create" these "Tags" manually,
if you have 100 videos, in theory, you must  "Create" :
-100 Female "Performers"
-100 Male "Performers"
-100 "Studio"
-500 "Tags"
if a tag requires about 20 seconds to "Create", "Apply", and "Save"
then, you need 8000 Seconds (2hours:13minutes:20seconds) for "Create", "Apply", and "Save" all 100 videos

**Comment by DogmaDragon** [comment, marked as answer]
You can use Identify task to auto create performers, studios, and tags. https://docs.stashapp.cc/in-app-manual/tasks/identify/

Tagger is a manual to allow people to verify information before they apply it.

**Comment by kucingkembar** [comment]
> You can use Identify task to auto create performers, studios, and tags. https://docs.stashapp.cc/in-app-manual/tasks/identify/
> 
> Tagger is a manual to allow people to verify information before they apply it.

thank you for the reply,
i start now, 
for the result it may take some hours to done,
i will report you the result later

**Comment by kucingkembar** [comment]
update : 
sorry for really late reply, 
because it contain some wrong data,
i need to clean all stash data and re "Phashing" all my videos,
my PC quite old, it take ages to process,

your method is right, 
thank you very much
